src/api/app/components/bs_request_overview_avatars_component.rb

```diff
@@ -13,14 +13,26 @@ def avatar_objects
     @avatar_objects ||= if @review.for_user?
                           [@review.user]
                         elsif @review.for_group?
-                          [@review.group.users, @review.group].flatten
+                          group_avatar_objects
                         elsif @review.for_package?
-                          @review.package.project.users
+                          package_avatar_objects
                         elsif @review.for_project?
-                          @review.project.users
+                          project_avatar_objects
                         end
   end
 
+  def group_avatar_objects
+    [@review.group.users, @review.group].flatten
+  end
+
+  def package_avatar_objects
+    [@review&.package&.project&.users].flatten.compact
+  end
+
+  def project_avatar_objects
+    [@review&.project&.users].flatten.compact
```

Comment:
The first `&` isn't needed, since the `project` association is always defined:

```suggestion
    [@review.project&.users].flatten.compact
```

Reply:
corrected